### ISSUE TITLE
Backlinks not working, "No refs in empty workspace"

### ISSUE THREAD

**Issue by coffenbacher** [kortina/vscode-markdown-notes#43]
_Troubleshooting: Before filing a new issue, please try to verify if this is a workspace / configuration issue, or an extension / os issue:_

- Checkout [the repo](https://github.com/kortina/vscode-markdown-notes): `git clone git@github.com:kortina/vscode-markdown-notes.git`
- Open the test notebook directory: `cd vscode-markdown-notes/test && code .`
- Open `top.md` in the test workspace in VS Code
- [ ] typing `[[t`,  _triggerSuggest_ (`⌃Space` by default) gives completions: `test.md`, `top.md`
- [ ] typing `[[s`,  _triggerSuggest_ (`⌃Space` by default) gives completions: `sub.md`, `sub2.md`
- [ ] typing `#a`,  _triggerSuggest_ (`⌃Space` by default) gives completions: `#another_tag`, `#tag`
- [ ] typing `[[test.md]]`, moving cursor over this word,  _peekDefinition_ (`⌥F12` by default) opens Peek Definition window
- Did your bug occur in the test directory?
  - [X] Yes
  - [ ] No 

**Describe the bug**

Backlinks panel is blank, and I get these errors:

![image](https://user-images.githubusercontent.com/245867/86036899-6b808e00-b9f3-11ea-9ede-1a5d4f2d8f20.png)

I couldn't find anything useful under the Extension Host logs, or any other category.

Any ideas on where to look next?

Thanks!

**Versions**
 - OS: [Ubuntu 20.04 LTS]
 - VS Code Version: 
```
Version: 1.47.0-insider
Commit: 9bf6af04dcb5c2e531cedca83cd188939ca1b0db
Date: 2020-06-26T18:55:32.092Z
Electron: 8.3.3
Chrome: 80.0.3987.165
Node.js: 12.13.0
V8: 8.0.426.27-electron.0
OS: Linux x64 5.4.0-33-generic snap
```

 - Extension Version: [v0.0.9]

**Comment by kortina** [comment]
Hmm, I am not entirely sure, tbh. I wonder if this is specific to Linux. I will try to debug a bit tn and let you know if I turn up any leads.

**Comment by coffenbacher** [comment]
Sounds good, thank you. I'm not experienced with extension development, but if it's not immediately obvious what the issue is after looking at it, I can see if I can run the source and try to get more debug information.

**Comment by kortina** [comment]
Hey, quick question. Did you open the workspace by doing the following?

```
cd vscode-markdown-notes/test && code .
```

? Cause if you open a single file, eg, `code test.md` and **don't** have a workspace open, you will see that message (which is expected)

OR, if you have the workspace open but there is no markdown file in an active editor, you will also see that message.

But if you open the workspace and then open `test.md` you should see the following files in the explorer:

![](https://s3.amazonaws.com/4rk/screen-shot-2020-06-29-19.43.35-ezr4lk2b.png)

If you are still having issues, can you take a screenshot that with this file / workspace open and put in a comment?

**Comment by coffenbacher** [comment]
Yep, that's how I open workspaces as well. :+1: 

The weirdest thing is that when I opened my computer this morning, the Backlinks are working perfectly now.

My two theories:
(1) something about the installation on my machine required a full restart of VSCode. I'm not sure I fully closed all instances to test, I'm so used to extensions working immediately it didn't even cross my mind.
(2) there was a passing incompatibility in VSCode Insiders which got fixed. This is my new version...somehow Electron & Chrome & V8 got downgraded major versions? Maybe they rolled back due to an issue there. 
```
Version: 1.47.0-insider
Commit: da73d2d11540232a5c75788ac0c9b41c3d432b2d
Date: 2020-06-30T05:36:03.333Z
Electron: 7.3.2
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Linux x64 5.4.0-33-generic snap
```

Anyways, given that I can no longer repro, I will close this issue for now. If it comes back for anyone, maybe they can cross-reference with my experience to narrow down the issue.
